lib/style_parser/filterer/index.js

```diff
@@ -20,16 +20,30 @@ var css = require('css');
 function filter(cssInput, cb) {
   // Remove HTML comment start/end
   cssInput = removeHtmlComments(cssInput);
+  cssInput = handleParsingErrors(cssInput);
```

Comment:
`handleParsingErrors` already parses the sheet, is there a way to avoid doing that a second time?